cmd/meeting-api/eventing/invite_accepted_subscriber.go

```
@@ -90,7 +93,17 @@ func (s *InviteAcceptedSubscriber) handle(msg *natsgo.Msg) {
 	s.wg.Add(1)
 	defer s.wg.Done()
 
-	ctx, cancel := context.WithTimeout(s.ctx, inviteAcceptedCallTimeout)
+	msgCtx := otel.GetTextMapPropagator().Extract(context.Background(), natsHeaderCarrier(msg.Header))
+	msgCtx, span := tracer.Start(msgCtx, "nats.process",
```

golangci-lint (2.12.2)

[error] ineffectual assignment to msgCtx

(ineffassign)